ocaml/parsing/jane_syntax_parsing.ml

```diff
@@ -244,8 +244,9 @@ module Embedded_name : sig
   val components : t -> components
 
   (** Convert one of these Jane syntax names to the embedded string form used in
-      the OCaml AST as the name of an extension node or an attribute; not
-      exposed. *)
+      the OCaml AST as the name of an extension node or an attribute; exposed
+      for extensions that only uses [Embedded_name] instead of the whole
+      infrastructure in this module, such as the dummy argument extension *)
```

Comment:
Yeah it should be moved to `mli`. 

I think it might be beneficial to explain why it's exposed, because similar to `needs_extra_checks`, this is a irregularity in `Jane_syntax`.